text/052-platform-release-cycle.md

```diff
@@ -0,0 +1,137 @@
+- Title: Coq and Coq platform release cycle
+
+- Drivers: Enrico
+
+----
+
+# Summary
+
+We aim at a release process for Coq and the Coq platform which makes it
+easier for users to provide feedback and for developers to deliver fixes.
+
+In a nutshell the Coq "beta period" is replaced by a Coq platform "beta period".
+Coq devs tag a almost final version on top of which the Coq platform core is
+built. Coq point release are then used as a *quick* mean to deliver fixes
+to the platform.
+
+# Motivation
+
+The beta testing period does not provide enough feedback to sensibly improve
+Coq between the `VX+beta1` and `VX.0` tags. The reasons are, most important
+last:
+- users have better things to do, frankly
+- one month is short, especially with Christmas/summer holydays around
+- in order to test Coq you are likely to need a bunch of Coq packages, and they
+  are typically only available after the beta period is over
+
+Moreover the "beta period" is also seen by devs as a time frame where they can
+still change many things. As a results users are even less happy to spend time
+working on a moving target.
+
+Finally, due to the cost of backporting breaking changes from the master
+branch to the release branch, during 8.12 and 8.13 release cycle this only
+happened very early, just after branching (only 2 in the beta period for 8.12,
+0 for 8.13).
+
+## New actor: platform
+
+The Coq platform is a selection of Coq package versions which work well
```

Comment:
which -> that ?

Reply:
You could also write just "Coq packages" and to avoid the redundancy with the versions in the next line.